<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>项目名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProjectNameEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProjectAddressEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>项目地址：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProjectAddressBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>181</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>选择项目地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>添加资源：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddFolderBtn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>360</y>
     <width>91</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QListWidget" name="FolderListWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>531</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SubFolderBtn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>420</y>
     <width>91</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBtn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>720</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBtn_2">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>720</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
